x-pack/plugins/ml/public/application/components/controls/select_severity/select_severity.tsx

```diff
@@ -124,23 +124,34 @@ export const SelectSeverity: FC<Props> = ({ classNames } = { classNames: '' }) =
   return <SelectSeverityUI severity={severity} onChange={setSeverity} />;
 };
 
-export const SelectSeverityUI: FC<{
-  classNames?: string;
-  severity: TableSeverity;
-  onChange: (s: TableSeverity) => void;
-}> = ({ classNames = '', severity, onChange }) => {
+export const SelectSeverityUI: FC<
+  Omit<EuiSuperSelectProps<string>, 'onChange' | 'options'> & {
+    classNames?: string;
+    severity: TableSeverity;
+    onChange: (s: TableSeverity) => void;
+  }
+> = ({ classNames = '', severity, onChange, compressed }) => {
   const handleOnChange = (valueDisplay: string) => {
     onChange(optionValueToThreshold(optionsMap[valueDisplay]));
   };
 
+  const options = useMemo(() => {
+    return getSeverityOptions();
+  }, []);
+
   return (
     <EuiSuperSelect
+      prepend={i18n.translate('xpack.ml.explorer.severityThresholdLabel', {
+        defaultMessage: 'Severity threshold',
```

Comment:
I'd vote to just label this control as `Severity`. It would then match the one for the swim lanes.

Reply:
Renamed in 452343d224f17d3a5fbaf3b0e7bcd918bd35effa